### DISCUSSION TITLE
Good!

### DISCUSSION THREAD

**Discussion by kakashi0x0** [community/community#14078] [Other Feature Feedback, Questions, & Ideas]
Absolutely loving the `For you` feed! Discovered new things.....

**Comment by irezaul** [comment]
Always updataing & Bringing new features makes a lot of work easier for us...

**Comment by jfullstackdev** [comment]
much more

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
